nats.go

```diff
@@ -2507,6 +2507,19 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// Servers returns the list of known servers, including additional
```

Comment:
@derekcollison Is it OK to return urls as they are (I can update the documentation to say user passed credentials will be present), or should we tag/filter?